
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-orangepi-4-lts.dts b/arch/arm64/boot/dts/rockchip/rk3399-orangepi-4-lts.dts
--- a/arch/arm64/boot/dts/rockchip/rk3399-orangepi-4-lts.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-orangepi-4-lts.dts
@@ -1237,10 +1237,6 @@
 	status = "okay";
 };
 
-&hdmi_in_vopl {
-	status = "disable";
-};
-
 &dp_in_vopb {
 	status = "disable";
 };

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-orangepi-4.dts b/arch/arm64/boot/dts/rockchip/rk3399-orangepi-4.dts
--- a/arch/arm64/boot/dts/rockchip/rk3399-orangepi-4.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-orangepi-4.dts
@@ -1181,10 +1181,6 @@
 	status = "okay";
 };
 
-&hdmi_in_vopl {
-	status = "disable";
-};
-
 &dp_in_vopb {
 	status = "disable";
 };